lib/collection/src/update_handler.rs

```diff
@@ -336,10 +336,10 @@ impl UpdateHandler {
                             ) {
                                 // Perform some actions when optimization if finished
                                 Ok(num_points_optimized) => {
-                                    // FIXME: Decide on ordering
-                                    // ToDo: Only increment if it's an indexing optimization
-                                    total_indexed_points
-                                        .fetch_add(num_points_optimized, Ordering::Relaxed);
+                                    if optimizer.name() == "indexing" {
+                                        total_indexed_points
+                                            .fetch_add(num_points_optimized, Ordering::Relaxed);
+                                    }
```

Comment:
This is incorrect. All optimizer types may index points, so we don't want to consider the optimizer type here.

I believe we want to look at the end result instead and probe whether any points were actually indexed.